docs/provider-integration/07-implementation-order.md

```
@@ -0,0 +1,188 @@
+**Why third:** Near-clone of LM Studio. If LM Studio works, this should work with a small set of tweaks.
+
+---
+
+### Milestone 4 · NVIDIA NIM (the complex one)
+
+Before starting, **verify the AI SDK supports `providerOptions.openai.body`** for arbitrary extras (the `ai-sdk-verifier` task). If yes, proceed. If no, switch to the fetch-interception fallback (see `03-nvidia-nim.md`).
+
+1. Create `src/lib/providers/nvidiaNim.ts` per `03-nvidia-nim.md`
+2. Add registration + barrel export
+3. Validate base case:
+   ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)